.github/workflows/gradle.yml

```diff
@@ -9,31 +9,22 @@ on:
 
 jobs:
   build:
-    strategy:
-      matrix:
-        java: [1.8, 11]
-        os: [ubuntu-latest, windows-latest, macos-latest]
-    runs-on: ${{ matrix.os }}
-    env:
-      gradle_version: 6.8.3 # set to empty to build with most recent version of gradle
-      gradle_commands: build # default is build
-    steps:
-      - uses: actions/checkout@v2
-      - name: Set up JDK ${{ matrix.java }}
-        uses: actions/setup-java@v1
-        with:
-          java-version: ${{ matrix.java }}
-      - name: Cache Gradle packages
-        uses: actions/cache@v2
-        with:
-          path: ~/.gradle/caches
-          key: ${{ runner.os }}-gradle-${{ hashFiles('**/*.gradle') }}
-          restore-keys: ${{ runner.os }}-gradle
-      - name: Wrap with specified version
-        run: gradle wrapper --gradle-version=${{ env.gradle_version }}
-        if: ${{ env.gradle_version != '' }}
-      - name: Wrap without version
-        run: gradle wrapper
-        if: ${{ env.gradle_version == '' }}
-      - name: Run commands
-        run: ./gradlew ${{ env.gradle_commands }}
+    uses: ome/action-workflows/.github/workflows/gradle_build.yml@v1
+  publish_snapshots:
+    if: startsWith(github.event.head_commit.message, 'Merge pull request')
```

Comment:
push even will also trigger on forked repo if you have credentials set up.

Reply:
Also realised this might get every push get deployed including on potential integration branches. Is this the expectation? Or are we limiting it to `master` and/or the `ome ` repo for now?

Reply:
I was planning to limit to ``master`` and ``ome`` but we can reconsider

Reply:
Works for me as step 1, we might need to re-use something like https://github.com/ome/bioformats/blob/7974737e8e8a5d1462b6d614a266e4a0f108009e/.github/workflows/maven.yml#L36